rpc/client/local/rpc_test.go

```diff
@@ -0,0 +1,183 @@
+package local_test
```

Comment:
also, this testing suite is duplicated in 2 places. I understand we want to test both implementations. Not sure what is the best way to deal with this.